### ISSUE TITLE
cant find any pokestops

### ISSUE THREAD

**Issue by hytvszz** [NoxxDev/NecroBot#657]
Hi it just happen this morning after bot all night, it says no pokestops nearby and ask if the area is set too small, but I did not change anything, I thought it is softban but I try other accounts and got same warning, 

is this IP ban? I have try different WIFI, so could it be IP ban or any other stuff?

btw the bot also not doing anything else but keep warning no pokestops

**Comment by GummiBoat** [comment]
Yes that's an IP-Ban. Just change it, and you're good to go. They usually last only 1-2 hours, so no need to worry.
